test/pub_update_test.dart

```diff
@@ -7,6 +7,7 @@ import 'package:process/process.dart';
 import 'package:pub_updater/pub_updater.dart';
 import 'package:test/test.dart';
 
+import 'fixtures/pre_release_package_info_response.dart';
 import 'fixtures/valid_package_info_response.dart';
```

Comment:
```suggestion
import 'fixtures/fixtures.dart';
```

Should a barrel file exists here?